Source/Relativity.Test.Helpers/Relativity.Test.Helpers.Example.NUnit/TestTemplate.cs

```diff
@@ -49,13 +49,13 @@ public class TestTemplate
 
 		#region TestfixtureSetup
 
-		[TestFixtureSetUp]
+		[OneTimeSetUp]
 		public void Execute_TestFixtureSetup()
 		{
 			//Setup for testing
 			var helper = new TestHelper(ConfigurationHelper.ADMIN_USERNAME, ConfigurationHelper.DEFAULT_PASSWORD);

```

Comment:
can you rename the variable to testHelper for easy reading of the code. helper is very generic.